Header header

float32 P_rate
float32 Q_rate
float32 R_rate

float32 X_acc
float32 Y_acc
float32 Z_acc

float32 roll
float32 pitch
float32 yaw

float32 latitude
float32 longitude
float32 altitude

float32 NS_speed
float32 EW_speed
float32 DU_speed
